modules/ROOT/pages/index.adoc

```diff
@@ -1,48 +1,96 @@
-It can be used as an alternative to {cass-migrator} ({cass-migrator-short}).
+svg::sideloader:astra-migration-toolkit.svg[]
 
-* xref:sideloader:sideloader-overview.adoc[{sstable-sideloader}]: {sstable-sideloader} is a service running in {astra-db} that directly imports data from snapshot backups that you've uploaded to {astra-db} from an existing {cass-reg}, {dse}, or {hcd} cluster.
+++++
+</div>
+++++
 
-////
-* https://github.com/datastax/zdm-proxy-automation[{product-automation}] repo for Ansible-based {product-proxy} automation.
-//{product-automation} 2.3.0, which enables ansible scripts and terraform to work with both Ubuntu and RedHat-family Linux distributions.
+[.[&>h2]:!hidden]
+== {empty}
 
-* https://github.com/datastax/dsbulk-migrator[{dsbulk-migrator}] repo for migration of smaller data quantities.
+[subs="macros,attributes"]
+++++
 
-* https://github.com/datastax/cassandra-data-migrator[{cass-migrator}] repo for migration of larger data quantities and where detailed verifications and reconciliation options are needed.
-////
+<h2 class="discrete !text-h1 !mt-12 !mb-6">Migration tools</h2>
 
-////
-Possible related content:
+<div class="grid gap-6 lg:grid-cols-4">
+  <div class="grid gap-4">
 
-https://docs.datastax.com/en/dse/6.8/tooling/migration-path-dse.html
+    svg:ROOT:icons/astra-db-cloud-backup-restores.svg[role="mx-auto max-w-xs md:mx-0 lg:max-w-none"]
 
-https://docs.datastax.com/en/dse/6.9/tooling/migration-path-dse.html
+    <h3 class="discrete !text-h2 !m-0">{product} ({product-short})</h3>
 
-MC only:
+    <p>Comprised of {product-proxy}, {product-utility}, and {product-automation}, you can continue to run your current application and migrate data from the origin to the target database without any downtime. {product-proxy} helps to manage the activity in transition.</p>
 
-https://docs.datastax.com/en/mission-control/migrate/oss-cass-to-mission-control.html
+    <div class="landing-a">
+        xref:ROOT:introduction.adoc[Get started with {product-short}]
+    </div>
 
-https://docs.datastax.com/en/mission-control/migrate/dse-to-mission-control.html
-////
\ No newline at end of file
+  </div>
+  <div class="grid gap-4">
+
+    svg:ROOT:icons/loading-data-into-astra-db.svg[role="mx-auto max-w-xs md:mx-0 lg:max-w-none"]
+
+    <h3 class="discrete !text-h2 !m-0">{cass-migrator} ({cass-migrator-short})</h3>
+
+    <p>You can use {cass-migrator-short} with {product-proxy} for zero-downtime migrations, or you can use {cass-migrator-short} alone for migrations with some downtime. You can also use {cass-migrator-short} for data validation.</p>
+
+    <div class="landing-a">
+        xref:ROOT:cdm-overview.adoc[Get started with {cass-migrator-short}]
+    </div>
+
+  </div>
+  <div class="grid gap-4">
+
+    svg:ROOT:icons/migrating-apps.svg[role="mx-auto max-w-xs md:mx-0 lg:max-w-none"]
+
+    <h3 class="discrete !text-h2 !m-0">{dsbulk-migrator}</h3>
```

Comment:
Why do you list DSBulk Migrator as an option here? It's not in the left nav. You have DSBulk Loader in the left nav, but not as a choice here. It's confusing.

Reply:
Yes it is confusing. I have been going over the docs and Jira to figure it out over the last few weeks, and I need to switch the DSBulk Loader section to be about DSBulk Migrator. 

https://datastax.jira.com/browse/DOC-5192

Do you want me to add that to this ticket or fast follow in another ticket?

Reply:
@KimberlyFields Here is the new preview. I have addressed the discrepancy with DSBulk Loader vs DSBulk Migrator. 

https://d5rxiv0do0q3v.cloudfront.net/doc-5183/data-migration/index.html